### ISSUE TITLE
Provide a way to view/update write quota in the UI

### ISSUE THREAD

**Issue by jrhee17** [line/centraldogma#612]
It would be nice if we can view/modify write quota per project/repository in the UI

**Comment by ikhoon** [comment]
Indeed! Please use REST API or edit `dogma/metadata.json` as a workaround. https://github.com/line/centraldogma/blob/082d87bd172fc0705579f2774458466e0dbfc1c6/server/src/main/java/com/linecorp/centraldogma/server/internal/api/MetadataApiService.java#L292-L306
